flang/include/flang/Lower/TransformationalRuntime.h

```diff
@@ -0,0 +1,27 @@
+//===-- Lower/TransformationalRuntime.h -- lower transformational intrinsics==//
```

Comment:
The `--*- C++ -*-===//` part of this line (which is missing) is more important than the comment "lower transformational intrinisics" which seems self-evident/redundant.